common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/wizards/TestProjectConfigurationWizardPage.xtend

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.dsl.common.ui.wizards
+
+import org.eclipse.jface.wizard.WizardPage
+import org.eclipse.swt.SWT
+import org.eclipse.swt.events.SelectionEvent
+import org.eclipse.swt.events.SelectionListener
+import org.eclipse.swt.layout.GridData
+import org.eclipse.swt.layout.GridLayout
+import org.eclipse.swt.widgets.Button
+import org.eclipse.swt.widgets.Combo
+import org.eclipse.swt.widgets.Composite
+import org.eclipse.swt.widgets.Label
+import org.eclipse.swt.widgets.List
+import org.testeditor.dsl.common.ui.utils.Constants
+
+class TestProjectConfigurationWizardPage extends WizardPage {
```

Comment:
public mehtods should have return types (there is also an option in Eclipse where we create warnings for this case)